src/Tribe/Views/V2/Customizer.php

```diff
@@ -373,16 +373,74 @@ public function filter_global_elements_css_template( $css_template, $section, $c
 				.tribe-block__organizer__website a,
-				.tribe-events-pro .tribe-events-pro-venue__meta-website a,
 				.tribe-block__venue__website a,
 				.tribe_events p a {
 					color: <%= global_elements.link_color %>;
 				}
 			';
 		}
 
+		if ( $customizer->has_option( $section->ID, 'background_color' ) ) {
+			$css_template .= '
+				.tribe-common.tribe-events-view,
+				.tribe-events-page-template,
+				.single-tribe_organizer {
+					background-color: <%= global_elements.background_color %>;
+				}
+			';
+		}
+		
+		if ( $customizer->has_option( $section->ID, 'event_title_color' ) ) {
+			// Event Title overrides.
+			$css_template .= '
+				.single-tribe_events .tribe-events-single-event-title,
+				.tribe-events-view .tribe-events-calendar-list__event-title,
+				.tribe-events-view a.tribe-events-calendar-list__event-title-link,
+				.tribe-events-view .tribe-events-calendar-month__calendar-event-title,
+				.tribe-events-view a.tribe-events-calendar-month__calendar-event-title-link,
+				.tribe-events-view .tribe-events-calendar-month__multiday-event-bar-title,
+				.tribe-events-view .tribe-events-calendar-month-mobile-events__mobile-event-title,
+				.tribe-events-view a.tribe-events-calendar-month-mobile-events__mobile-event-title-link,
+				.tribe-common .tribe-events-calendar-month__calendar-event-tooltip-title,
+				.tribe-common a.tribe-events-calendar-month__calendar-event-tooltip-title-link,
+				.tribe-common a.tribe-events-calendar-month__calendar-event-tooltip-title-link:hover,
+				.tribe-common a.tribe-events-calendar-month__calendar-event-tooltip-title-link:focus,
+				.tribe-events-view .tribe-events-calendar-day__event-title,
+				.tribe-common a.tribe-events-calendar-day__event-title-link,
+				.tribe-common .tribe-events-calendar-day__event-title-link:hover,
+				.tribe-common .tribe-events-calendar-day__event-title-link:focus {
+					color: <%= global_elements.event_title_color %>;
+				}
+				
+				.tribe-common .tribe-events-calendar-list__event-title-link:hover,
+				.tribe-common .tribe-events-calendar-list__event-title-link:focus,
+				.tribe-common .tribe-events-calendar-month__calendar-event-title-link:hover,
+				.tribe-common .tribe-events-calendar-month__calendar-event-title-link:focus,
+				.tribe-common a.tribe-events-calendar-month__calendar-event-tooltip-title-link:hover,
+				.tribe-common a.tribe-events-calendar-month__calendar-event-tooltip-title-link:focus,
+				.tribe-common .tribe-events-calendar-day__event-title-link:hover,
+				.tribe-common .tribe-events-calendar-day__event-title-link:focus  {
+					border-color: <%= global_elements.event_title_color %>;
+				}
+			';
+		}
+		
+		if ( $customizer->has_option( $section->ID, 'event_date_time_color' ) ) {
+			// Event Date Time overrides.
+			$css_template .= '
+				.single-tribe_events .tribe-events-schedule h2,
+				.tribe-events-calendar-list__event-datetime,
+				.tribe-events-calendar-day__event-datetime,
```

Comment:
I know we're trying to keep specificity down, but we also want to keep our styles targeted, so let's add `.tribe-events` before these 2.